LayoutTests/accessibility/gtk/mathml-underover-scripts.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+</head>
+<body>
+<div id="content">
+  <!-- https://w3c.github.io/mathml-aam/#mathml-element-mappings -->
+  <math>
+    <munder id="munder" class="test">
+      <mtext>BASE</mtext>
+      <mtext>UNDER</mtext>
+    </munder>
+    <mover id="mover" class="test">
+      <mtext>BASE</mtext>
+      <mtext>OVER</mtext>
+    </mover>
+    <munderover id="munderover" class="test">
+      <mtext>BASE</mtext>
+      <mtext>UNDER</mtext>
+      <mtext>OVER</mtext>
+    </munderover>
+  </math>
+</div>
+<p id="description"></p>
+<div id="console"></div>
+<script>
+    description("This tests the ATK roles of MathML underover elements.");
```

Comment:
But we don't use ATK anymore, so I guess the term to use now must be "AT-SPI role."

Reply:
Same comment applies to the commit message. You've mapped it to Atspi::Role::Section, not ATK_ROLE_SELECTION.